actions/get_node_id.py

```diff
@@ -25,5 +25,8 @@ def run(self, node=None):
         self.connect()
 
         orion_data = self.get_node(node)
+        if not orion_data.npm_id:
```

Comment:
couldn't we just return `(False, 'No Nodes match {}')` instead of raising? this will still cause it to error out

Reply:
Switched to Return